azure-mgmt-redis/azure/mgmt/redis/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "3.0.0"
+VERSION = "2.0.0"
```

Comment:
The version number is generated by a configuration file that was incorrect. Will be fixed of course :)